solr_conf/conf/solrconfig.xml

```diff
@@ -191,20 +191,10 @@
     </autoSoftCommit>
 
   </updateHandler>
-  <requestHandler name="/admin/" class="org.apache.solr.handler.admin.AdminHandlers" />
```

Comment:
I believe solr is quite good in this regards. This stuff is deprecated in version 5. And removed in version 6. It is not needed anymore.